src/SearchUsers/SearchUsers.css

```
@@ -0,0 +1,19 @@
+.note{
+    text-align: center;
+}
+
+.displayData{
+    margin: 0 1rem 0 1rem;
+    /* Not sure why align and boreders don't work here, come back to it
+    align-items: center;
+    align-self: center; */
+}
+
+.userDataContainer{
+    color: black;
+    /* Not sure why align and boreders don't work here, come back to it
+    align-items: center;
+    align-self: center;
+    align-content: center; */
+
+}
```

Couple of notes - let's remove the comments (we'll leave alignment to Grid elems from MUI for later); make sure the file has a newline at the end (note the red circle); make sure to follow convention when adding CSS class names - I've updated them here to follow convention; since the classes here are all the same specificity, let's list them alphabetically to make them easier to read; the `margin` prop in `.display-data` can be simplified

```suggestion
.display-data {
    margin: 0 1rem 0 1rem;
}

.note {
    text-align: center;
}

.user-data-container {
    color: black;
}

```